src/components/docs-anchor/__tests__/docs-anchor.test.tsx

```diff
@@ -25,8 +25,9 @@ describe('DocsAnchor', () => {
       </CurrentProductProvider>
     )
 
-    const result = screen.queryByText('This is a link')
+    const result = screen.queryByRole('link')
```

Comment:
This change was needed because the markup of this component changes to have a `<span>` within the rendered `<a>`. When doing `queryByText`, the `<span>` is the `result`. Doing `queryByRole` gets us the `<a>` element.

This is a good long-term change in case the mark up of this component changes in the future. If for some reason an element other than `<a>` is rendered, then this query will ensure that the role of the element is properly set.